Framework/Algorithms/test/AnnularRingAbsorptionTest.h

```diff
@@ -43,11 +43,11 @@ class AnnularRingAbsorptionTest : public CxxTest::TestSuite {
     MatrixWorkspace_sptr outWS = alg->getProperty("OutputWorkspace");
     TS_ASSERT(outWS);
 
-    const double delta(1e-08);
+    const double delta(1e-04);
```

Comment:
It is a lot of variation. I had to change it as a direct result of the algorithm receiving different random numbers from the new Mersenne Twister algorithm, and the ones it now receives happen to produce a lot more variation than before. I'm not honestly sure what to do about this as the AnnularRingAbsorption algorithm itself is producing such extreme variation.